src/c++/perf_analyzer/genai-perf/genai_perf/export_data/csv_exporter.py

```diff
@@ -65,14 +63,17 @@ def __init__(self, config: ExporterConfig):
         self._args = config.args
 
     def export(self) -> None:
-        csv_filename = self._output_dir / DEFAULT_OUTPUT_DATA_CSV
-        logger.info(f"Generating {csv_filename}")
-
-        with open(csv_filename, mode="w", newline="") as csvfile:
-            csv_writer = csv.writer(csvfile)
-            self._write_request_metrics(csv_writer)
-            csv_writer.writerow([])
-            self._write_system_metrics(csv_writer)
+        filename = str(self._args.profile_export_file)
+        if filename.endswith(".json"):
+            filename = filename[:-5]
+        filename += "_genai_perf.csv"
```

Comment:
I think we can use `Path.stem` to extract the name of a file.
```suggestion
        filename = f"{self._args.profile_export_file.stem}_genai_perf.csv"
```

Reply:
Thanks for bringing this up! For CSV, this works if you update it to:
```python
filename = self._output_dir / f"{self._args.profile_export_file.stem}_genai_perf.csv
```

However, I wanted to keep it consistent with the JSON exporter. The JSON exporter extracts the args into a dict and, for some reason, then stem is unavailable due to the resulting object being a string (strings do not have stem available).This seems the clearest/best way to do it consistently for both exporters. At least for my PR, I want it to be consistent, and it doesn't seem to be the best use of time now to see why the exporters use different data structures for args to standardize them right now.

Reply:
Ah I see. Thanks for explaining 👍 It is strange that stem does not work in json exporter...
If it's because the `self._args.profile_export_file` is string, could we use `os.path.splitext` and `os.path.basename` ([source](https://stackoverflow.com/a/678242)) to extract the filename? I just don't want to leave the `[:5]` indexing in the code.

For example,
```python
prefix = os.path.splitext(os.path.basename(self._args.profile_export_file))[0]
filename = f"{prefix}_genai_perf.csv"
```
This will allow us to avoid some problems like potential user misspelling json in the filename (e.g. if user types `custom_export.jso`, then the filename will be `custom_expor_genai_perf.csv`).

Reply:
It's a string. Apologies for not clarifying earlier.

Great suggestion, done!